static/app/components/dropdownMenu/index.tsx

```diff
@@ -75,6 +75,10 @@ interface DropdownMenuProps
    * Pass class name to the outer wrap
    */
   className?: string;
+  /**
+   * Optionally add more props to be passed on to the useOverlay hook.
+   */
+  dropdownOverlayProps?: UseOverlayProps;
```

Comment:
Adding this in to get access to the dropdown's `onOpenChange` event handler. This is used to expose the dropdown's `open/close` state outside the component (in my case, to add/remove CSS classes for the expected behavior)

Reply:
i'd go in favor of using the same pattern as these props just to keep things the same and https://github.com/getsentry/sentry/blob/14b245fa5a6d3cbe5470690be282d448ea8a0c80/static/app/components/dropdownMenu/index.tsx#L56-L67